Example/Sources/CustomChatViewController.swift

```diff
@@ -66,6 +60,38 @@ class CustomChatViewController: ChatViewController {
             .disposed(by: disposeBag)
     }
     
+    func updateNavigationBar() {
+        title = presenter?.channel.name
+        updateNavigationBar(withChannelImage: nil)
+        
+        if let imageURL = presenter?.channel.imageURL {
+            DispatchQueue.global().async {
+                guard let imageData = try? Data(contentsOf: imageURL),
+                    let image = UIImage(data: imageData)?.resized(targetSize: .init(width: 44, height: 44))?.original else {
+                        return
+                }
+                
+                DispatchQueue.main.async {
+                    self.updateNavigationBar(withChannelImage: image)
+                }
+            }
+        }
+    }
+    
+    func updateNavigationBar(withChannelImage channelImage: UIImage?) {
+        var items: [UIBarButtonItem] = [membersCountButton]
+        
+        if let channelImage = channelImage {
+            items.append(UIBarButtonItem(image: channelImage, style: .plain, target: nil, action: nil))
+        }
+        
+        if (navigationController?.viewControllers.count ?? 1) > 1 {
+            navigationItem.rightBarButtonItems = items
+        } else {
```

Comment:
Why there is a v2 change in a v3 PR? 🤯

Reply:
I need to get a sample JSON and add this to v2 Example app. I think it useful to keep it if we need to get more JSON data like that.

Reply:
I see. I think you should've created a separate PR for this change. It's totally our of context.